modules/ding_nodelist/js/carousel.js

```diff
@@ -13,21 +13,23 @@
           }
         });
 
-        $(this).find('.ding_nodelist-items').slick({
-          nextArrow: '<i class="icon-next"></i>',
-          prevArrow: '<i class="icon-prev"></i>',
-          autoplay: true,
-          speed: 500,
-          autoplaySpeed: delay,
-          responsive: true,
-          dots: true,
-          infinite: true,
-          slidesToScroll: 1,
-          slidesToShow: 1,
-          customPaging: function(slick, index) {
-            return '<a>' + (index + 1) + '</a>';
-          }
-        });
+        if ($(this).find('.ding_nodelist-items').hasClass('slick-initialized') === false) {
+          $(this).find('.ding_nodelist-items').slick({
```

Comment:
Is there a reason why you do not use [`:not()`](https://api.jquery.com/not-selector/) instead of running the query twice?

```js
$(this).find('.ding_nodelist-items:not(.slick-initialized)').slick({...});
```

Reply:
Fixed.